packages/cozy-client/src/models/index.js

```diff
@@ -1,6 +1,7 @@
-export { triggerResults } from './triggers'
```

Comment:
This could be considered a breaking change, but on the other hand `triggerResults` is undefined so I'm quite confident no one was using it. Some trigger methods are used in cozy-banks but they import straight from `models/triggers`.